README.md

```diff
@@ -126,6 +126,13 @@ await apiValidator.initialize() // Note that for a large number of specs this ca
 const validationResult = apiValidator.validateLiveRequestResponse(requestResponsePair)
 ```
 
+### Regression testing
+
+Output of the OAV tool has been snapshotted and committed to the repo. The regression test may be run on a sample or all of https://github.com/azure/azure-rest-api-specs. If there are changes to the snapshots the build produces a git patch file as an artifact which may be used to update the snapshots.
+
+[Fast Regression (~10mins)](https://dev.azure.com/azure-sdk/public/_build?definitionId=166&_a=completed)

```

Comment:
the fast one will run as a merge validation correct ?

Reply:
yes